### PR TITLE
2.10.0 changelog

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+Bugfixes
+--------
+- Fix a potential incorrect length for Spooled sets during fallback.
+  (`#1937 <https://github.com/vertexproject/synapse/pull/1937>`_)
+- Fix an issue with the Telepath ``Client`` object caching their ``Method``
+  and ``GenrMethod`` attributes across re-connections of the underlying
+  ``Proxy`` objects.
+  (`#1939 <https://github.com/vertexproject/synapse/pull/1939>`_)
+  (`#1941 <https://github.com/vertexproject/synapse/pull/1941>`_)
+- Fix a but where a temporary spool slab cleanup failed to remove all
```
Comment (mikemoritz) [inline CHANGELOG.rst:104]: typo "fix a but"

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
 *****************
 
 
+v2.10.0 - TBD
```
Comment (mikemoritz) [inline CHANGELOG.rst:8]: PR name is 2.1.0 instead of 2.10.0

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+  privilege escalation to execute (in order, for instance, to create nodes).
+  Services using this mechanism may use the ``storm.asroot.cmd.<<cmd name>>``
+  hierarchy to grant this permission.
+  (`#1953 <https://github.com/vertexproject/synapse/pull/1953>`_)
+- Add ``$lib.json`` StormTypes Library to convert between string data and
+  primitives.
+  (`#1949 <https://github.com/vertexproject/synapse/pull/1949>`_)
+- Add a ``parallel`` command to allow for executing a portion of a Storm
+  query in parallel. Add a ``background`` command to execute a Storm query
+  as a detached task from the current query, capturing variables in the.
```
Comment (Cisphyx) [inline CHANGELOG.rst:44]: Extra period here

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+  process.
+  (`#1931 <https://github.com/vertexproject/synapse/pull/1931>`_)
+  (`#1957 <https://github.com/vertexproject/synapse/pull/1957>`_)
+- Add a ``$lib.exit()`` function to StormTypes to allow for quickly
+  exiting a Storm query.
+  (`#1931 <https://github.com/vertexproject/synapse/pull/1931>`_)
+- Add ``$lib.bytes.upload()`` to Stormtypes for streaming bytes into the
```
Comment (Cisphyx) [inline CHANGELOG.rst:51]: The T in StormTypes isn't capitalized here

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+- Add Storm commands to manage locking and unlocking deprecated model
+  properties.
+  (`#1909 <https://github.com/vertexproject/synapse/pull/1909>`_)
+- Add ``cron.cleanup`` command to make it easy to clean up completed cron
+  jobs.
+  (`#1942 <https://github.com/vertexproject/synapse/pull/1942>`_)
+- Add date of death properties and consistently named photo secondary
+  properties.
+  (`#1929 <https://github.com/vertexproject/synapse/pull/1929>`_)
+- Add model additions for repsentings education and awards.
```
Comment (Cisphyx) [inline CHANGELOG.rst:63]: 'repsentings'

File: CHANGELOG.rst [hunk 6]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+  together.
+  (`#1935 <https://github.com/vertexproject/synapse/pull/1935>`_)
+- Allow the Storm Service cmdargs to accept any valid model type in the
+  ``type`` value.
+  (`#1923 <https://github.com/vertexproject/synapse/pull/1923>`_)
+  (`#1936 <https://github.com/vertexproject/synapse/pull/1936>`_)
+- Add ``>``, ``<``, ``>=`` and ``<=`` comparators for ``inet:ipv4`` type.
+  (`#1938 <https://github.com/vertexproject/synapse/pull/1938>`_)
+- Add configuration options to the Axon to limit the amount of data which
+  can be stored in it. Add a configuration option the the Cortex to limit
```
Comment (Cisphyx) [inline CHANGELOG.rst:91]: 'the the Cortex'

File: CHANGELOG.rst [hunk 7]
```diff
@@ -5,6 +5,115 @@ Synapse Changelog
...
+  (`#1937 <https://github.com/vertexproject/synapse/pull/1937>`_)
+- Fix an issue with the Telepath ``Client`` object caching their ``Method``
+  and ``GenrMethod`` attributes across re-connections of the underlying
+  ``Proxy`` objects.
+  (`#1939 <https://github.com/vertexproject/synapse/pull/1939>`_)
+  (`#1941 <https://github.com/vertexproject/synapse/pull/1941>`_)
+- Fix a but where a temporary spool slab cleanup failed to remove all
+  files from the filesystem that were created when the slab was made.
+  (`#1940 <https://github.com/vertexproject/synapse/pull/1940>`_)
+- Move exceptions which do not sublcass ``SynErr`` out of ``synapse/exc.py``.
```
Comment (Cisphyx) [inline CHANGELOG.rst:107]: 'sublcass'

File: CHANGELOG.rst [hunk 8]
```diff
@@ -5,6 +5,111 @@ Synapse Changelog
...
+- Add support for adding extended forms to the Cortex. This allows users to
+  define their own forms using the existing types which are available in the
+  Synapse data model.
+  (`#1944 <https://github.com/vertexproject/synapse/pull/1944>`_)
+- The Storm ``and`` and ``or`` statements now short-circuit and will return
+  when their logical condition is first met. This means that subsequent
+  clauses in those statements may not be executed.
+  (`#1952 <https://github.com/vertexproject/synapse/pull/1952>`_)
+- Add a mechanism for Storm Services to specify commands which may require
+  privilege escalation to execute (in order, for instance, to create nodes).
```
Comment (jnwatson) [inline CHANGELOG.rst:35]: plz "privilege elevation"

**Review by jnwatson — APPROVED** [review]

**Pull request merged**